integration/simulation/params/wallet_utils.go

```diff
@@ -11,6 +13,10 @@ import (
 	"github.com/obscuronet/go-obscuro/integration/datagenerator"
 )
 
+const (
+	faucetPrivateKeyHex = "8dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b" // The faucet's private key.
```

Comment:
let's move this to the faucet class as well.

Reply:
Ok. I didn't think it was worth the shared dependency.